instance/github.py

```diff
@@ -157,8 +164,39 @@ def __init__(self, number, fork_name, branch_name, title, username, body=''):
         self.body = body
 
     @property
+    def settings(self):
+        """
+        Settings contained in the PR body
+        """
+        return yaml.load(get_settings_from_pr_body(self.body)) or {}
+
+    @property
+    def database_url(self):
+        """
+        An optional database url, from the DATABASE_URL setting in the PR body
+        """
+        return self.settings.get(self.SETTINGS['DATABASE_URL'])
+
+    @property
+    def mongo_url(self):
+        """
+        An optional mongo url, from the MONGO_URL setting in the PR body
+        """
+        return self.settings.get(self.SETTINGS['MONGO_URL'])
+
+    @property
+    def ephemeral_databases(self):
+        """
+        The EPHEMERAL_DATABASES setting given in the PR body
+        """
+        return self.settings.get(self.SETTINGS['EPHEMERAL_DATABASES'])
+
+    @property
     def extra_settings(self):
         """
-        Extra settings contained in the PR body
+        Extra ansible settings from the PR body, as a yaml string
         """
-        return get_settings_from_pr_body(self.body)
+        if not self.settings:
+            return ''
+        return yaml.dump({key: value for key, value in self.settings.items()
+                          if key not in self.SETTINGS.values()})
```

Comment:
What is the rationale for this change?  Normalising the format?

Note that the new version will parse the body twice – one time for each invocation of `self.settings`.  It might be worthwhile to cache the settings once they got evaluated to avoid parsing them in every invocation of the property.